src/headless/converse-muc.js

```diff
@@ -368,11 +368,17 @@ converse.plugins.add('converse-muc', {
             },
 
             extractReference (text, index) {
-                for (let i=index; i<text.length; i++) {
+                for (let i = index; i < text.length; i++) {
```

Comment:
Please remove these spaces again. Check the style guide for info on how we use spaces.

https://conversejs.org/docs/html/style_guide.html